int16 command_L
int16 command_R
